src/store/index.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import rootReducer from './rootReducer';
```

Comment:
Let's maybe stick to import order: npm modules first and our custom modules second. Third are styles.

This is just a proposal